scripts/build/mkversioninfo

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env sh
```

Comment:
nit: any reason the file doesn’t have a `.sh` extension?

Reply:
To remain consistent with the current state of our scripts